file-formats/ints/type/zif_aff_ints_v1.intf.abap

```diff
@@ -0,0 +1,322 @@
+      "! This is the low value of modeling context filter
+      value            TYPE string,
+      "! <p class="shorttext">Modeling Context Filter High Value</p>
+      "! This is the high value of modeling context filter
+      high_value       TYPE string,
+    END OF ty_modelling_context_filters,
+
+    "! <p class="shorttext">Parameter defaults</p>
+    "! These are the scenario parameter defaults
+    BEGIN OF ty_parameter_defaults,
+      "! <p class="shorttext">UUID</p>
+      "! This is the scenario UUID
+      guid    TYPE c LENGTH 36,
+      "! <p class="shorttext">Parameter Name</p>
+      "! This is the parameter name
+      name    TYPE c LENGTH 256,
+      "! <p class="shorttext">Parameter Context</p>
+      "! This is the parameter context
+      context TYPE c LENGTH 1000,
+      "! <p class="shorttext">Parameter Value</p>
+      "! This is the parameter value
+      value   TYPE string,
+    END OF ty_parameter_defaults,
+
+    "! <p class="shorttext">Parameter Signature</p>
+    "! These are the scenario parameters signature
+    BEGIN OF ty_parameters,
+      "! <p class="shorttext">Parameter Name</p>
+      "! This is the parameter name
+      name        TYPE c LENGTH 256,
+      "! <p class="shorttext">Parameter Type</p>
+      "! This is the parameter type
+      type        TYPE c LENGTH 32,
+      "! <p class="shorttext">Parameter Description</p>
+      "! This is the parameter description
+      description TYPE string,
+      "! <p class="shorttext">Storage</p>
+      "! This is the storage type
+      storage     TYPE c LENGTH 100,
+      "! <p class="shorttext">Parameter Context Flag</p>
+      "! This is the flag for parameter context
+      context     TYPE c LENGTH 1,
+      "! <p class="shorttext">Parameter Size</p>
+      "! This is the size of parameter
+      param_size  TYPE i,
+    END OF ty_parameters,
+
+    "! <p class="shorttext">Additional Information</p>
+    "! These are the scenario additional information
+    BEGIN OF ty_additional_info,
+      "! <p class="shorttext">Additional Information Key</p>
+      "! This is an additional information key
+      additional_info_key TYPE c LENGTH 50,
+      "! <p class="shorttext">Additional Information Value ID</p>
+      "! This is the additional information value ID
+      value_id            TYPE i,
+      "! <p class="shorttext">Additional Information Value</p>
+      "! This is the value of an additional information
+      value               TYPE c LENGTH 1024,
+      "! <p class="shorttext">Additional Information Position ID</p>
+      "! This is the position of an additional information ID
+      position_id         TYPE i,
+    END OF ty_additional_info,
+
+    "! <p class="shorttext">Additional Information Texts</p>
+    "! These are the scenario additional information texts
+    BEGIN OF ty_additional_info_text,
+      "! <p class="shorttext">Language Key</p>
+      "! This is the language key
+      language            TYPE sy-langu,
```

Comment:
AFF usually does not contain multiple languages. It only represents the original language of the object. Translations will be stored in separate files.

```suggestion
```